MaksymBulbiak/HW8/Task_password.py

```diff
@@ -0,0 +1,32 @@
+import re
+
+"""
+Write a Python program to check the validity of a password (input from users).
+"""
+
+password = input("Password:\n")
+# try:
+#     assert  re.findall("[a-z]",password)
+#     assert  re.findall("[A-Z]",password)
+#     assert  re.findall("[$#@]",password)
+#     assert  re.findall("[0-9]",password)
+#     assert  len(password) < 16
+#     assert  len(password) > 6
+#     print("You have entered!")
+# except:
+#     print("At least 1 letter between [a-z].\nand 1 letter between [A-Z].\n\
+# At least 1 character from [$#@].\nAt least 1 number between [0-9].\n\
+# Minimum length 6 characters.\nMaximum length 16 characters.\n\
+# At least 1 number between [0-9].")
+
```

Comment:
good job
here is another implementation:
import re

password = input("Enter your password:\n")

if re.match("^(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.*[$#@])(?=.{6,16}$).*", password):
    print("Nice pass! Psss, I won't tell anyone ;)")
else:
    print("Password is not valid. Check if you have:"
          """
    At least 1 letter between[a-z] and 1 letter between[A-Z].
    At least 1 number between[0-9].
    At least 1 character from [$#@].
    Minimum length 6 characters.
    Maximum length 16 characters.
          """
           )